Tests/Editor/GameLiftPlugin/AnywherePageTests.cs

```diff
@@ -0,0 +1,176 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Amazon.GameLift.Model;
+using AmazonGameLift.Editor;
+using AmazonGameLiftPlugin.Core.ApiGatewayManagement;
+using AmazonGameLiftPlugin.Core.SettingsManagement.Models;
+using Moq;
+using NUnit.Framework;
+
+namespace Editor.Window.UnitTests
+{
+    [TestFixture]
+    public class AnywherePageTests
```

Comment:
The test class should mimic the file being tested. This appears to be testing the GameLiftRequestAdapter, not the AnywherePage editor window